react-list.es6

```diff
@@ -181,8 +204,9 @@ module.exports = class ReactList extends Component {
     const max = this.getScrollSize() - this.props.scrollParentViewportSizeGetter(this);
     const scroll = Math.max(0, Math.min(actual, max));
     const el = this.getEl();
-    this.cachedScrollPosition = this.getOffset(scrollParent) + scroll - this.getOffset(el);
-    return this.cachedScrollPosition;
+    const scrollPosition = this.getOffset(scrollParent) + scroll - this.getOffset(el);
+    this.cachedScrollData = [axis, scrollPosition];
```

Comment:
Let's do `{axis, scrollPosition}` for clarity. It's much nicer to see `cachedScrollData.axis` and `cachedScrollData.position` than `[0]` and `[1]`.